doc/source/cluster/kubernetes/examples/argocd.md

```diff
@@ -0,0 +1,461 @@
+    targetRevision: v1.4.0  # update this as necessary
+    path: helm-chart/kuberay-operator
+    helm:
+      skipCrds: true   # CRDs are already installed in Step 1
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: ray-cluster
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+      - CreateNamespace=true
+```
+
+Note the `skipCrds: true` setting in the Helm configuration. This is required because the CRDs were installed separately in Step 1.
+
+Apply the ArgoCD Application:
+
+```sh
+kubectl apply -f ray-operator.yaml
+```
+
+Wait for the operator Application to sync and become healthy. You can check the status using:
+
+```sh
+kubectl get application ray-operator -n argocd
+```
+
+Alternatively, if you have the ArgoCD CLI installed:
+
+```sh
+argocd app wait ray-operator
+```
+
+Verify that the KubeRay operator pod is running:
+
+```sh
+kubectl get pods -n ray-cluster -l app.kubernetes.io/name=kuberay-operator
+```
+
+## Step 3: Deploy a RayCluster
+
+Now deploy a RayCluster with autoscaling enabled and three different worker groups. Create a file named `raycluster.yaml` with the following content:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: raycluster
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: ray-cluster
+  ignoreDifferences:
+    - group: ray.io
+      kind: RayCluster
+      name: raycluster-kuberay
+      namespace: ray-cluster
+      jsonPointers:
+        - /spec/workerGroupSpecs/0/replicas
+        - /spec/workerGroupSpecs/1/replicas
+        - /spec/workerGroupSpecs/2/replicas
+  source:
+    repoURL: https://ray-project.github.io/kuberay-helm/
+    chart: ray-cluster
+    targetRevision: "1.4.1"
+    helm:
+      releaseName: raycluster
+      valuesObject:
+        image:
+          repository: docker.io/rayproject/ray
+          tag: latest
+          pullPolicy: IfNotPresent
+        head:
+          rayStartParams:
+            num-cpus: "0"
+          enableInTreeAutoscaling: true
+          autoscalerOptions:
+            version: v2
+            upscalingMode: Default
+            idleTimeoutSeconds: 600 # 10 minutes
+            env:
+            - name: AUTOSCALER_MAX_CONCURRENT_LAUNCHES
+              value: "100"
+        worker:
+          groupName: standard-worker
+          replicas: 1
+          minReplicas: 1
+          maxReplicas: 200
+          rayStartParams:
+            resources: '"{\"standard-worker\": 1}"'
+        additionalWorkerGroups:
+          additional-worker-group1:
+            image:
+              repository: docker.io/rayproject/ray
+              tag: latest
+              pullPolicy: IfNotPresent
+            disabled: false
+            replicas: 0
+            minReplicas: 0
+            maxReplicas: 30
+            rayStartParams:
+              resources: '"{\"additional-worker-group1\": 1}"'
```

Comment:
`Resources` is required here and so does `additional-worker-group2` below. Or, it might get the error that autoscaler couldn't be ready. https://github.com/ray-project/kuberay/issues/4211

```suggestion
            rayStartParams:
              resources: '"{\"additional-worker-group1\": 1}"'
            resources:
              requests:
                cpu: "1"
                memory: "1G"
```